internal/runners/packages/list.go

```diff
@@ -139,7 +139,7 @@ func fetchCheckpoint(commit *strfmt.UUID) (model.Checkpoint, *failures.Failure)
 		return nil, model.FailNoData.New(locale.T("package_no_data"))
 	}
 
-	return model.FilterCheckpointPackages(checkpoint), fail
+	return checkpoint, fail
```

Comment:
This change doesn't seem to be related to importing packages. This will also include requirements that are not packages in the output of `state packages`

Reply:
It's related since otherwise I cannot filter out the language. I'll address the second point.